### DISCUSSION TITLE
Nelson-Meads optimizer

### DISCUSSION THREAD

**Discussion by jaredbernard** [hahnlab/CAFE5#146] [Q&A] [answered]
Hi, @benfulton. Your README says there ought to be the options OPTIMIZER_STRATEGY_INITIAL_VARIANTS and OPTIMIZER_STRATEGY_PERTURB_WHEN_CLOSE, yet these are not parameters with flags for the command line. Am I missing something?

*Sorry, I obviously meant "Nelder-Mead" in the subject line.

**Comment by benfulton** [comment, marked as answer]
Those are options available at compile-time for the optimizer. We didn't make them command line parameters because we didn't notice any particular improvement in the optimizer when using them, but they are still available. If you want to try them out, edit **config.h** and add

`#define OPTIMIZER_STRATEGY_INITIAL_VARIANTS `

Or define the other one. Then rebuild the application and run the analysis.

**Reply by jaredbernard** [reply]
Thanks, I will try that!
